docs/src/main/asciidoc/security-authentication-mechanisms.adoc

```diff
@@ -217,6 +217,66 @@ The following properties can be used to configure form-based authentication:
 
 include::{generated-dir}/config/quarkus-vertx-http_quarkus.http.auth.adoc[opts=optional, leveloffset=+1]
 
+[[two-factor-auth]]
+==== Two-factor authentication
+
+The form-based authentication mechanism supports two-factor authentication (2FA) with a one-time authentication token second-factor option.
+
+.Enable two-factor authentication
+[source,properties]
+----
+quarkus.http.auth.form.authentication-token.enabled=true
+quarkus.http.auth.form.authentication-token.request-redirect-path=/authentication-token-form <1>
```

Comment:
This path can be made a default value, `quarkus.http.auth.form.authentication-token.enabled=true` should be enough for such a case. Also propose to shorten the property to `authentication-token.redirect-path`